CSharpMinifier.GUI/frmMain.cs

```diff
@@ -33,8 +33,14 @@ void FillList(string[] fileNames)
 			foreach (var fileName in fileNames)
 			{
 				var lbItem = new ListBoxItem(fileName, Path.GetFileName(fileName));
-				Sources[lbItem.Key] = File.ReadAllText(lbItem.Key);
-				lbInputFiles.Items.Add(lbItem);
+                try
+                {
+                    Sources[lbItem.Key] = File.ReadAllText(lbItem.Key);
+                    lbInputFiles.Items.Add(lbItem);
+                }
+                catch(Exception e)
+                { }
```

Comment:
@theCalcaholic can you please fix it in this pull request? :) I'll merge it then.